pkg/migration-tool/writer/writer.go

```diff
@@ -74,12 +88,24 @@ func (rw *RemoteWrite) Run(errChan chan<- error) {
 				if !ok {
 					return
 				}
-				blockRef.SetDescription("preparing to push", 1)
-				ts = timeseriesRefToTimeseries(blockRef.MergeProgressSeries(rw.progressTimeSeries))
-				buf = []byte{}
-				if err = rw.sendSamplesWithBackoff(context.Background(), ts, &buf); err != nil {
-					errChan <- fmt.Errorf("remote-write run: %w", err)
-					return
+				blockRef.UpdatePBarMax(blockRef.PBarMax() + rw.shardsSet.num)
+				{
```

Comment:
why this block?

Reply:
Same reason as https://github.com/timescale/promscale/pull/385#discussion_r551721431.